nsxt/resource_nsxt_policy_l2_vpn_session_test.go

```diff
@@ -207,13 +255,40 @@ func testAccNsxtPolicyL2VpnSessionCheckDestroy(state *terraform.State, displayNa
 	return nil
 }
 
-func testAccNsxtPolicyL2VpnSessionPreConditionTemplate() string {
-	return fmt.Sprintf(`
+func testAccNsxtPolicyL2VpnSessionPreConditionTemplate(isT0 bool) string {
+	resourceName := getAccTestResourceName()
```

Comment:
`resourceName` would still be different across different invocations of  `testAccNsxtPolicyL2VpnSessionPreConditionTemplate`, correct? This means that terraform will perform update for parent resources. My suggestion was to refer to a global variable that would only be initialized once.